backend/wordAlgorithm/word.py

```diff
@@ -0,0 +1,94 @@
+# Format of xml taken as input:
+#    <Words>
+#        <Word>
+#            <Language>Dutch</Language>
+#            <Word>ja</Word>
+#            <Translation>yes</Translation>
+#            <Category/>
+#            <Word_Difficulty/>
+#            <Extensions/>
+#        </Word>
+#    </Words>
+
+
+words = []
+
+# Word object
+class Word(object):
+    originalWord = "";
+    language = ""
+    translatedWord = "";
+    category = "";
+    wordValue = 0;
+    extensions = []
+    progress = 0;
+    wordDifficulty = 0;
+    
+# Word constructor
+def makeWord(language, originalWord, translatedWord, category, extensions, wordDifficulty):
```

Comment:
Again, dont use Camel Case instead use underscores. and please follow [this commenting convention](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).